### DISCUSSION TITLE
Feature request: New panel chrome UI - add option to center title

### DISCUSSION THREAD

**Discussion by szerwi** [grafana/grafana#67298] [Dashboards, Panels & Visualizations]
Hi,
Personally I find the new panel chrome UI much worse on my dashboard, where I have many small gauge and singlestat panels.
It it a bit odd to have panel centered, whereas the title is aligned to the left.
![1](https://user-images.githubusercontent.com/19311511/234620418-2f1b1be2-201d-4be4-beb2-4ba4f8b97a16.png)

I think there should be option to center the title, so it will be right above the panel itself, as it is in the old UI style:
![2](https://user-images.githubusercontent.com/19311511/234620692-ceccf6de-ea6a-422c-b11b-40fee0a44c43.png)

Do you plan to add the option to center the panel's title?

**Comment by caparisi** [comment]
Definitely a graphical must to have.

**Comment by KOCIU22** [comment]
Hello Title Name in Panel now is on left side not on middle like before can you add option to chose or back to middle setting by default

![image](https://user-images.githubusercontent.com/78568161/235137294-12bc9bd8-dba4-47ed-a91c-12bfb4710a2a.png)

**Reply by Igorgro** [reply]
Sorry for off-topic, but what font and style do you use for panel? It looks fantastic

**Reply by KOCIU22** [reply]
Hello

For style and background  i use Boom Theme PLugin and font name is Orbitron from Google :)

**Comment by zarch1972** [comment]
Agree with above.  Panel headings were in the centre in 9.4, but now they are shoved over to the left in 9.5

![image](https://user-images.githubusercontent.com/23335221/236466040-d852a078-a0ce-4f00-8cc1-2e53695278c5.png)


Can we have the option to choose centre please?

**Comment by natellium** [comment]
Hey everyone, thanks for your feedback. We would like to unify this conversation in one platform, in this case our forum https://community.grafana.com/t/the-panel-title-is-not-centered/85506/30

I will close this discussion, but please feel do add your feedback in the post linked.

**Comment by szerwi** [comment]
@natellium I've noticed that since Grafana 10.2.0 it is even not possible to disable new panel chrome UI and go back to the old header 
UI. Option to center titles hasn't been added even though it seems like many people are waiting for it.

What is the status of this feature request?
